security/docker/Dockerfile.node-agent

```diff
@@ -1,10 +1,11 @@
 # BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
 ARG BASE_DISTRIBUTION=default
 
+# Version is the base image version from the TLD Makefile
+ARG BASE_VERSION=latest
+
 # The following section is used as base image if BASE_DISTRIBUTION=default
-# No tag available https://hub.docker.com/_/scratch?tab=description
-# hadolint ignore=DL3006
-FROM scratch as default
+FROM docker.io/istio/base:${BASE_VERSION} as default
```

Comment:
I am not against this but this is a legit change, not just a incremental update. Should have security folks take a look (I think they will be required anyways). Same with citadel

Reply:
@howardjohn had indicated there was verbal approval from the security team for this change.

Reply:
I took a look at the BASE_VERSION, it looks OK to me.